app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/milk_background"
+    android:orientation="vertical"
+    tools:context="com.easy.cooking.bsccookingapp.LoginActivity">
+
+    <TextView
+        android:id="@+id/login_title_tv"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_margin="24dp"
```

Comment:
Please move 24dp to dimens.xml resource file. Create default margins/paddings resources to be used everywhere you need.

e.g.

<dimen name="fsSmallest">8sp</dimen>
    <dimen name="fsSmall">10sp</dimen>
    <dimen name="fsMedium">12sp</dimen>
    <dimen name="fsXMedium">14sp</dimen>
    <dimen name="fsLarge">16sp</dimen>
    <dimen name="fsXLarge">20sp</dimen>
    <dimen name="fsHuge">24sp</dimen>
    <dimen name="fsXHuge">32sp</dimen>

    <dimen name="paddingSmallest">2dp</dimen>
    <dimen name="paddingSmall">4dp</dimen>
    <dimen name="paddingDefault">8dp</dimen>
    <dimen name="paddingLarge">16dp</dimen>
    <dimen name="paddingHuge">32dp</dimen>

    <dimen name="marginSmallest">2dp</dimen>
    <dimen name="marginSmall">4dp</dimen>
    <dimen name="marginDefault">8dp</dimen>
    <dimen name="marginLarge">16dp</dimen>
    <dimen name="marginHuge">32dp</dimen>